airbyte-integrations/connectors/destination-clickhouse/metadata.yaml

```diff
@@ -26,5 +26,5 @@ data:
   ab_internal:
     sl: 200
     ql: 200
-  supportLevel: certified
```

Comment:
We need to also update ab_internal.sl accordingly (see https://www.notion.so/Internal-Metadata-Fields-32b02037e7b244b7934214019d0b7cc9?pvs=4#587617c9002243359322eae01ac0f150)